testing/test_driver.py

```diff
@@ -107,6 +107,56 @@ def test_pass(driver_kwargs):
     testdir.quick_qa("--driver", "Remote", file_test, passed=1)
 
 
+def test_https_host_port(testdir):
```

Comment:
@platonoff-dev You added a check for https and no protocol but shouldn't you also test the case for http protocol?

Reply:
I think `test_default_host_port` does the same... 
https://github.com/pytest-dev/pytest-selenium/blob/a91b1f5e0252bb10654516f3bb73598b11a72e22/testing/test_driver.py#L94

Reply:
No I don't think so that is another case where the protocol is not http:
host |  url   | tested |
------| ----| ------|
https://... | https://... | yes
http://... | http:// | no
no.protocol | http:// | yes
localhost | http:// | yes

Reply:
Yeah, I hurried with this a little

Reply:
No problem, I think this can be solved with a simple parametrization: http, https